### DISCUSSION TITLE
What does the similarity percentage of suggestions by machine translation mean?

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [WeblateOrg/weblate#7932] [Q&A] [answered]
Whether the string is untranslated or translated, there is always a similarity percentage for each suggestion by machine translation (not translation memory). What does it mean?
![image](https://user-images.githubusercontent.com/49396421/181863250-fcdad32b-f571-4daa-b7f7-1c62f4c0aea5.png)

**Comment by nijel** [comment, marked as answer]
It is currently hard-coded number to rank quality of the services. It is probably very inaccurate estimation of that, though....

**Reply by Geeyun-JY3** [reply]
Is it a required field?  The current hard-coded number under the similarity column may confuse or mislead users (such as me).

**Reply by nijel** [reply]
It is needed for ordering of results, but it indeed could be hidden. I've done that in dc8266f08def4b03070b17f31712afa8e702d67f
